app/posts/manage-teacher-training-applications/2020-05-04-setting-up-permissions-iteration-3.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Setting up permissions (iteration 3)
+description: Let providers set up permissions between themselves and their partner organisations
+date: 2020-05-04
+---
+
+{% from "figure/macro.njk" import appFigure with context %}
+
+Previous iterations:
+
+* [first iteration](/manage-teacher-training-applications/setting-up-permissions)
+* [second iteration](/manage-teacher-training-applications/setting-up-permissions-iteration-2)
+
+In this iteration we improved the design and tested it with a user who works at Somerset SCITT who:
+
+* run and ratify their own courses
+* ratify courses for Growing Expert Teachers, Taunton Teaching Alliance, The Beach Teaching School and Ventrus Teaching School Alliance.
+
+The prototype is based on the user's access on Publish which shows they belong to Ventrus Teaching School Alliance. During the session we learnt that the user is also belong of the other 3 organisations too.
+
+This means that while this person works for Somerset SCITT, these training providers delegate power and control to people outside of their organisation.
+
+This set up is quite common across other partnerships on Apply.
+
+The journeys include:
+
+- setting up permissions for partner organisations for the first time
+- inviting a new user to their organisation
+- managing organisational permissions and users
+
+## Hypotheses
+
+### Showing the user belongs to multiple organisations
+
+The [previous design](/manage-teacher-training-applications/setting-up-permissions-iteration-2) didn’t handle the situation where a user belongs to multiple organisations.
+
+If we show the user they're part of multiple organisations
+Then they'll understand what’s about to happen on the following screens
+
+{{ appFigure({
+  image: {
+    file: "before-you-start.png"
+  }
+}) }}
+
+{{ appFigure({
+  image: {
+    file: "data-sharing.png"
```

Comment:
Suggest a crop of this screenshot, showing just the top and bottom?

The intro paragraph somewhat hides the multi-org aspect.